### DISCUSSION TITLE
Why I don't get the Galaxy brain badge?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#41802] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Profile Badges Feedback

### Hi, I have the galaxy brain and bronze galaxy brain badge, but why I don't get the silver badge?

**If you click on your profile pic up on right an then click on your discussions**

![Screenshot 2022-12-15 092457](https://user-images.githubusercontent.com/112943652/207800215-dc243c16-ba60-4ccb-a404-b2f80f954e34.png)

**and then add these tags:**

![Screenshot 2022-12-15 092529](https://user-images.githubusercontent.com/112943652/207800272-24e0cbb1-8d43-4776-99a5-f2e0a92d9977.png)

and even that, check only those that I have answered **WITHOUT**: Those who start with MitkoVtori/reponame, those that I created.
**I still have 16+ answers.**

**Comment by mohamedzhioua** [comment, marked as answer]
hello brother @MitkoVtori , i just tried to see how many answers you got by adding (commenter:MitkoVtori is:answered ) but it only show  answer :( maybe someone deleted his discussion or something wrong  anyway i hope you get  the x4

**Comment by jfullstackdev** [comment]
I think it's frozen now, the top 1 here with more than 200 answers for sure, is not receiving the tiers, I'm the top 2 at this time with constant marked answers for several months and my tier did not update,

**Reply by MitkoVtori** [reply]
Ouch, your hurts even more. It stays just at one. YOu answered at least 8 **MY questions** 😕

**Reply by DSDmark** [reply]
Ya, Mr.jdevstatic is actually talking about @airtower-luna. she is top on ```Most helpful Last 30 days```, but even this. Her badge is still the same, I mention Mrs.airtower-luna here. So, maybe she knows something about it.

![image](https://user-images.githubusercontent.com/85744816/207864458-5d5582ba-c342-446e-8346-3f995b06a2b9.png)

**Reply by mohamedzhioua** [reply]
@MitkoVtori @DSDmark  you will get your Galxy brain just when your answers are accepted , but github will also count  how many times you participate when your participate in a discussion but another one get an accepted answer . if the disscussion has now accepted answer it will not count for u . anyway you get rewarded just for accepted answers

**Reply by mohamedzhioua** [reply]
@MitkoVtori  @DSDmark  to understand me you should take a look on ((commenter:MitkoVtori is:answered ) (in search ))
you will see that  @MitkoVtori  appear in all disscussions that has accepted answer even if it's not you who has the accepted one

**Reply by DSDmark** [reply]
@mohamedzhioua, Ok, its satisfy the MitkoVtori condition. But, what about airtower-luna.

**Reply by mohamedzhioua** [reply]
i agree with you @DSDmark  , yeah it s weird in airtower-luna case , but for @MitkoVtori  he has only 9 accepted answers so it's fine for him

**Reply by MitkoVtori** [reply]
> @MitkoVtori @DSDmark to understand me you should take a look on ((commenter:MitkoVtori is:answered ) (in search )) you will see that @MitkoVtori appear in all disscussions that has accepted answer even if it's not you who has the accepted one

I know, As I mentioned In the message **ONLY THOSE That I've answered**
Even If we remove the discussions that ain't mines, start with MitkoVtori/, I still have 18+(ONLY ME SET AS ANSWER)

**Reply by MitkoVtori** [reply]
> i agree with you @DSDmark , yeah it s weird in airtower-luna case , but for @MitkoVtori he has only 9 accepted answers so it's fine for him

Wait what? It shows me I have at least 16(without those that I am author of, repos that I own), and I know it shows everything I have commented, but even that, after a check I have 18. I don't understand what Is going on.

**Reply by mohamedzhioua** [reply]
@MitkoVtori  so it show u more than it shows me , aha i see now  ...dont worry bro , it will be fine , im sure , everyone is facing the same issue
and github will fix it soon .

**Reply by airtower-luna** [reply]
I suspect there's some issue with answers imported from the old community forum messing with the count, but it never bothered me enough to file a bug report. If you do I'd be curious about the reply though.

**Comment by MitkoVtori** [comment]
@github-staff Why is this happening? How can I get my silver badge?
also check https://github.com/orgs/community/discussions/41802#discussioncomment-4408835

**Comment by MitkoVtori** [comment]
Well, I don't know what the bug was, but I just got the badge on my 23th answered discussion(I think)
I don't know if it's fixed, or just smth happened, but another 16 ain't problem for me, even though I should of had more.

**Reply by mohamedzhioua** [reply]
@MitkoVtori congratulation brother , really happy for you , wishing you get the gold one , and wishing the silver for me :=)

**Reply by MitkoVtori** [reply]
> @MitkoVtori congratulation brother , really happy for you , wishing you get the gold one , and wishing the silver for me :=)

Hope you get it soon! 🙏

**Reply by mohamedzhioua** [reply]
@MitkoVtori accept this one to help me ;)

**Reply by MitkoVtori** [reply]
> @MitkoVtori accept this one to help me ;)

This is a Quote reply on my comment, so I can't accept it, but I'll accept your [official comment](https://github.com/orgs/community/discussions/41802#discussioncomment-4408165) :) <3
